### ISSUE TITLE
Inline source maps being request with path

### ISSUE THREAD

**Issue by auchenberg** [stacktracejs/stacktrace-gps#17]
This change https://github.com/stacktracejs/stacktrace-gps/commit/20042831f6a3f8aed19abd9e98bdc1e42b6ef82f#diff-a14450dbe613721f5b4d8361cc0e5ec0R244 has broken the inline source map support, as it causes the inline-source map to be fetched from the network with the base url prefixed.

Actual:
 sourceMappingURL: ``"http://localhost:7000/sourcemapsexample/build/data:application/json;base64,eyJ2Z``

Expected:
sourceMappingURL : ``"data:application/json;base64,e"``

**Comment by auchenberg** [comment]
Change could be:
```
          var sourceMappingURL = _findSourceMappingURL(source)
          var isDataUrl = sourceMappingURL.substr(0, 5) === 'data:'

          if (sourceMappingURL[0] !== '/' && !isDataUrl) {
            sourceMappingURL = fileName.substring(0, fileName.lastIndexOf('/') + 1) + sourceMappingURL
          }
```

**Comment by eriwen** [comment]
Fixed by #18
